templates/expo-template-bare-minimum/ios/Podfile

```
@@ -21,7 +24,7 @@ target 'HelloWorld' do
   use_react_native!(
     :path => config[:reactNativePath],
     :hermes_enabled => podfile_properties['expo.jsEngine'] == 'hermes',
-    :fabric_enabled => flags[:fabric_enabled],
+    :fabric_enabled => flags[:fabric_enabled] || podfile_properties['expo.newArchEnabled'],
```

```suggestion
    :fabric_enabled => flags[:fabric_enabled],
```

we don't have to change this. the react_native_pods.rb will based on the `RCT_NEW_ARCH_ENABLED` to setup the flags.

---

Ohh nice I thought we had to update both `ENV['RCT_NEW_ARCH_ENABLED']` and `:fabric_enabled `